web/src/app/settings/items/page.tsx

```diff
@@ -0,0 +1,48 @@
+"use client"
+
+import { Box, Button } from '@mui/material'
+import React from 'react'
+import { container } from "./style"
+import CustomTable from '@/components/Table'
+import { useHooks } from './hooks'
+import CustomModal from '@/components/Modal'
+
+const ItemPage = () => {
+  const {
+    columnSchema,
+    dataSource,
+    fields,
+    handleSubmit,
+    isOpenModal,
+    onSubmit,
+    handleCreateItem,
+    handleCloseModal,
+    isEditMode
```

Comment:
delete     isEditMode,
```suggestion

    btnName,
```